stellarphot/settings/models.py

```diff
@@ -545,27 +564,81 @@ class PhotometryOptions(BaseModelWithTableRep):
     fwhm_by_fit: bool = True
 
 
+class PassbandMapEntry(BaseModel):
+    """
+    long loing liong liong liong long long
```

Comment:
What is this?

Reply:
oops, meant to edit that 😬 it was keyboard mashing.